### DISCUSSION TITLE
When do we plan to release the latest master change

### DISCUSSION THREAD

**Discussion by houhaodong1122** [nielsbasjes/yauaa#430] [Q&A] [answered]
Hi.

  We have a project which use this lib to parse user agent to get device info. but we found this lib have issue for Galaxy S10 SC-03L device user agent. I have raised this issue: https://github.com/nielsbasjes/yauaa/issues/422 last week. and this has been fixed.
  I want to know when do we plan release the latest master change. and we can upgrade the lib version which contain this fixing.

Best Regards

**Comment by nielsbasjes** [comment]
I'll get a new version out in a few days.
Also I recommend using something like Renovate or Dependabot to automate your dependency updating.

**Comment by nielsbasjes** [comment, marked as answer]
Released 6.8
Do note that the new caching implementation needs Java 11.

**Reply by houhaodong1122** [reply]
> Released 6.8 Do note that the new caching implementation needs Java 11.

Thank you.
